packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 26.1.5
+
+* [kotlin] Fixes a "bridge method" warning that occurs when a class uses `*PigeonEventChannelWrapper` as a
+  supertype without implementing all of its methods.
```

Comment:
Is the "without implementing all of its methods" part right? The video_player code *was* implementing both of these methods.